.github/workflows/deploy-xtgeo-pypi.yml

```diff
@@ -8,10 +8,29 @@ env:
   CIBW_BEFORE_ALL_LINUX: "sh scripts/swig_manylinux.sh"
   CIBW_BEFORE_ALL_WINDOWS: "choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
   CIBW_BEFORE_ALL_MACOS: "brew install swig"
-  CIBW_TEST_COMMAND: 'python -c "import xtgeo; print(xtgeo.__version__)"'
+  CIBW_BEFORE_TEST: |
+    pushd {project}
+    pip install -r requirements/requirements.txt
+    pip install -r requirements/requirements_test.txt
+
+  # 3.9 have trouble with install pytables on Mac and Win, due to missing wheel
+  # so full testing on those platforms are postponed.
```

Comment:
But doesn't this mean that tests will only be partial on all python versions, not only 3.9?

Reply:
Will refactor setup